packages/react/src/components/Debug/utils.ts

```diff
@@ -0,0 +1,106 @@
+import * as _ from 'lodash'
+
+export const includes = (s, target) => _.toLower(s).indexOf(_.toLower(target)) !== -1
```

Comment:
Can you please set input types on these functions?

Reply:
Added typings and comments. These helper functions are required because lodash does not provide equivalents to these (either does not handles nested objects, or it's not working entirely the way we need). Let me know if you have better proposal for some of them.